### PR TITLE
cronjobs to explicitely use python3

### PR DESCRIPTION
**Pull request by lbesnard**

### CODE REVIEW

File: lib/python/IMOSnetCDF.py [hunk 1]
```diff
@@ -104,12 +104,12 @@ def close(self):
         self.date_created = time.strftime('%Y-%m-%dT%H:%M:%SZ', time.gmtime())
         if not DEBUG: self.deleteEmptyAttributes()
         self._F.close()
-        if self.__dict__.has_key('tmpFile'):
+        if 'tmpFile' in self.__dict__:
             # rename to desired filename and set permissions
             move(self.tmpFile, self.filename)
-            os.chmod(self.filename, 0644)
+            os.chmod(self.filename, 0o644)
```
Comment (bpasquer) [inline lib/python/IMOSnetCDF.py]: pls correct the typo in the mode

Comment (bpasquer) [reply lib/python/IMOSnetCDF.py]: sorry my bad, didn't know it is the way to code the chmod .

File: lib/python/IMOSnetCDF.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
```
Comment (mhidas) [inline lib/python/IMOSnetCDF.py]: Actually, this is a module, so doesn't really need this line.

Comment (lbesnard) [reply lib/python/IMOSnetCDF.py]: absolutely. I removed most shebangs from modules. I must have not think for that one

**Pull request merged**